### ISSUE TITLE
jsxstyle/preact broken ≥2.0.9

### ISSUE THREAD

**Issue by rperce** [smyte/jsxstyle#101]
Upgrading jsxstyle to or past 2.0.9 breaks `jsxstyle/preact` imports.

With 2.0.8 installed,
```
$ ls node_modules/jsxstyle/preact
jsxstyle-preact.cjs.js  jsxstyle-preact.d.ts  jsxstyle-preact.es.js  package.json
```

With 2.0.9 installed, however,
```
$ ls node_modules/jsxstyle/preact
jsxstyle-preact.amd.js  jsxstyle-preact.cjs.js  jsxstyle-preact.d.ts  jsxstyle-preact.es.js
```

and I see NPM error that it can't find any of
* `jsxstyle/preact{,.js,.json}
* `jsxstyle/preact/index{,.js,.json}
* `jsxstyle/preact/package.json`

(which makes sense, since none of those exist).

**Comment by meyer** [comment]
@rperce thanks, looking now…

**Comment by rperce** [comment]
Holy cow, 2 minute response time is amazing! You rock :+1:

**Comment by meyer** [comment]
Fixed in 93128ed and published as `jsxstyle@2.0.11`. Sorry for the trouble. I was being overly clever when I updated the `.npmignore` file for `jsxstyle` a few commits ago: https://github.com/smyte/jsxstyle/commit/e379385a8407734da66c3e8e3fdf42d67932f5e2#diff-5e3bdc8ec31c3acdd9b8559a74eec87fL3

package.json is bac[k 🎉🎉🎉](http://gunshowcomic.com/comics/20141111.png) https://unpkg.com/jsxstyle@2.0.11/preact/package.json

**Comment by meyer** [comment]
I’ve since added a publish manifest test that ensures that the expected package files are published. Should prevent this issue from occurring in the future.

the test: [`tests/meta.spec.ts#L54-L72`](https://github.com/smyte/jsxstyle/blob/cc2773d/tests/meta.spec.ts#L54-L72)
the snapshot: [`tests/__snapshots__/meta.spec.ts.snap#L24-L110`](https://github.com/smyte/jsxstyle/blob/cc2773d/tests/__snapshots__/meta.spec.ts.snap#L24-L110)
